ssl/d1_lib.c

```diff
@@ -656,16 +923,15 @@ int DTLSv1_listen(SSL *ssl, BIO_ADDR *client)
                  * We treat invalid cookies in the same was as no cookie as
                  * per RFC6347
                  */
-                next = LISTEN_SEND_VERIFY_REQUEST;
+                next = LISTEN_SEND_COOKIE_REQUEST;
             } else {
                 /* Cookie verification succeeded */
                 next = LISTEN_SUCCESS;
             }
         }
 
-        if (next == LISTEN_SEND_VERIFY_REQUEST) {
+        if (next == LISTEN_SEND_COOKIE_REQUEST) {
             WPACKET wpkt;
-            unsigned int version;
```

Comment:
Please move the declaration of the wpkt inside the construct_ functions unless there is a good reason to keep it here.